<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的音乐</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/image/ios8_Music_32px_1173465_easyicon.net.png</normaloff>:/image/image/ios8_Music_32px_1173465_easyicon.net.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QDialog#Dialog
{

    border:1px solid #5F5F5F;
	
	border-radius:0px;
}*/</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>350</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
border-radius:3px;
	border-image: url(:/image/image/background/bg4.jpg);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
    QSlider::add-page:Horizontal
     {     
background-color: rgba(255, 255, 255,100);
        height:2px;  
     } 
     QSlider::sub-page:Horizontal   
    {  
background-color: rgb(255, 255, 255);
        height:2px;
     }
    QSlider::groove:Horizontal   
    { 
        background:transparent; 
        height:2px;
    }  
    QSlider::handle:Horizontal  
    { 
        height:2px; 
        width:14px; 
	
	border-image: url(:/image/image/圆形 (9).png);
  margin: -6 0px; 
    }
      
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Comic Sans MS&quot;;
color: rgb(162, 162, 162);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image: url(:/image/image/min.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;
background-size: 20px 20px;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image: url(:/image/image/quit.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;
background-size:20px 20px;

}
QPushButton:hover{

background-color: rgba(230,0,0,0.5);
}
QPushButton:pressed{
background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image: url(:/image/image/image/next.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-image: url(:/image/image/image-hover/next-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image/next.png);
background-repeat:no-repeat;
background-position:center center;
background-size:20px 20px;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>48</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
background-size:24px 24px;

}

QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
background-size:20px 20px;

}

QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;
background-size:20px 20px;

}

QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Vladimir Script&quot;;
color: rgb(167, 167, 167);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>50</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image: url(:/image/image/image/pre.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/image/image/image-hover/pre-hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image/pre.png);
background-repeat:no-repeat;
background-position:center center;
background-size:20px 20px;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
